MessageMedia.Api.Console/Program.cs

```diff
@@ -114,6 +113,8 @@ public static void SendScheduledMessage()
 
                 var result = client.SendScheduledMessage("", recipient1, message, messageId, messageScheduledDateTime);
                 DisplaySendMessageResult(result);
+                
+                DeleteScheduledMessages(new List<uint>{ messageId });
```

Comment:
I don't think this is a good idea given that it's within an example where you Send the scheduled message.
Probably better to include a separate example where it schedules then deletes.